### PR TITLE
feat: add ffi for scroll; add auto-release for tag

### PR DESCRIPTION
**Pull request by Lawliet-Chan**
Refer https://github.com/scroll-tech/scroll/pull/303

1. You can push a  git tag  and it will automatically create and upload the released package. ( I have already create the alpha-v1.0 )   
2. Move rust ffi codes of scroll here.

### CODE REVIEW

**Comment by lispc** [comment]
looks good for me. would you suggest merge it?

**Comment by Lawliet-Chan** [comment]
> looks good for me. would you suggest merge it?

Yes, will merge it after @HAOYUatHZ and you approve it.

**Review by lispc — APPROVED** [review]

**Review by HAOYUatHZ — APPROVED** [review]

**Pull request merged**
